runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -609,6 +609,15 @@ static void jitHookInitializeSendTarget(J9HookInterface * * hook, UDATA eventNum
       TR_VerboseLog::writeLineLocked(TR_Vlog_PERF, "Setting count=%d for %s", count, buffer);
       }
 
+#if defined(J9VM_OPT_MICROJIT)
+   if(optionsJIT->_mjitEnabled)
+      {
+      int32_t mjitCount = optionsJIT->_mjitInitialCount;
+      if(count)
```

Comment:
This replaces the count for an AOT load with the microjit count. Was that the intent?
If AOT loads continue to exist under microjit, then we should preserve the counts set for them.